pkg/specgen/namespaces.go

```diff
@@ -54,7 +54,7 @@ const (
 
 	// DefaultKernelNamespaces is a comma-separated list of default kernel
 	// namespaces.
-	DefaultKernelNamespaces = "cgroup,ipc,net,uts"
+	DefaultKernelNamespaces = "ipc,net,uts"
```

Comment:
Why dtop cgroup from this?

Reply:
Though I posted this fix because values in `--share` and a result of pod inspection are inconsistent, I'm not sure if it's worth sharing a cgroup namespace in a pod.Containers in a pod have their own cgroups under a pod cgroup by default.
If a container enters the cgroup namespace of the infra container like other shared namespaces, the container cannot see its own cgroup.Because this behavior doesn't seem so useful, I dropped 'cgroup' from the default shared namespaces.

It also causes me to do it that crun cannot put a container into a specified cgroup namespace currently (containers/crun#781).

Reply:
I would rather wait until crun is updated and retain the cgroup sharing by default, if possible.

Reply:
I agree, lets remove the change and retest with latest crun.

Reply:
Sure. I will remove this change.